paasta_tools/check_chronos_jobs.py

```diff
@@ -42,8 +52,14 @@ def compose_monitoring_overrides_for_service(chronos_job_config, soa_dir):
     monitoring_overrides['check_every'] = '1m'
     monitoring_overrides['runbook'] = monitoring_tools.get_runbook(
         monitoring_overrides, chronos_job_config.service, soa_dir=soa_dir)
+
+    if 'realert_every' not in monitoring_overrides:
+        guessed_realert_every = _guess_realert_every(chronos_job_config)
+        if guessed_realert_every is not None:
```

Comment:
@solarkennedy: I'm not sure I understand why they will have to wait another 24h before the alert goes off?

check_chronos_jobs has check_every = 1m.
The realert_every logic is only applied to the 'create' event (https://github.com/Yelp/sensu_handlers/blob/master/files/base.rb#L241).

It looks like it should work as expected i.e. the alert will goes off as soon as check_chronos_jobs sends OK.